lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -2,17 +2,35 @@
 Tests for users API
 """
 import datetime
+import ddt
+from django.conf import settings
 from django.utils import timezone
+from django.template import defaultfilters
+from mock import patch
+import pytz
```

Comment:
Reorder these imports to be:
* Standard library imports, alphabetized
* Line break
* 3rd party library imports (so django.*), alphabetized
* Line break